<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="WorkSpace" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>881</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="Legend" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>881</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="Levels" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>881</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionSmooth_shading"/>
    <addaction name="actionGrid"/>
    <addaction name="actionIsolines"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="actionSmooth_shading"/>
   <addaction name="actionGrid"/>
   <addaction name="actionIsolines"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSmooth_shading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Smooth shading</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionIsolines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Isolines</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WorkSpace</class>
   <extends>QWidget</extends>
   <header location="global">workspace.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Legend</class>
   <extends>QWidget</extends>
   <header location="global">legend.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Levels</class>
   <extends>QWidget</extends>
   <header>levels.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
